src/pudl/package_data/eia860/column_maps/cooling_equipment.csv

```diff
@@ -34,16 +34,15 @@ tower_type_3,,,,,tower_type_3,tower_type_3,tower_type_3,tower_type_3,tower_type_
 tower_water_rate_100pct_gallons_per_minute,,,,,tower_water_rate_gallons_per_minute,tower_water_rate_gallons_per_minute,tower_water_rate_gallons_per_minute,tower_water_rate_gallons_per_minute,tower_water_rate_gallons_per_minute,tower_water_rate_gallons_per_minute,tower_water_rate_gallons_per_minute,tower_water_rate_gallons_per_minute,tower_water_rate_gallons_per_minute,tower_water_rate_gallons_per_minute
 tower_water_rate_100pct_cubic_feet_per_second,tower_water_rate_pct,tower_water_rate_pct,tower_water_rate_pct,tower_water_rate_cubic_feet_per_second,,,,,,,,,,
-power_requirement_mw,,,,power_requirement_mw,power_requirement_mw,power_requirement_mw,power_requirement_mw,power_requirement_mw,power_requirement_mw,power_requirement_mw,power_requirement_mw,power_requirement_mw,power_requirement_mw,power_requirement_mw
-power_requirement_kwh,power_requirement_kwh,power_requirement_kwh,power_requirement_kwh,,,,,,,,,,,
-cost_total_thousand_dollars,cost_total,cost_total,cost_total,cost_total_thousand_dollars,cost_total_thousand_dollars,cost_total_thousand_dollars,cost_total_thousand_dollars,cost_total_thousand_dollars,cost_total_thousand_dollars,cost_total_thousand_dollars,cost_total_thousand_dollars,cost_total_thousand_dollars,cost_total_thousand_dollars,cost_total_thousand_dollars
-cost_ponds_thousand_dollars,cost_ponds,cost_ponds,cost_ponds,cost_ponds_thousand_dollars,cost_ponds_thousand_dollars,cost_ponds_thousand_dollars,cost_ponds_thousand_dollars,cost_ponds_thousand_dollars,cost_ponds_thousand_dollars,cost_ponds_thousand_dollars,cost_ponds_thousand_dollars,cost_ponds_thousand_dollars,cost_ponds_thousand_dollars,cost_ponds_thousand_dollars
-cost_towers_thousand_dollars,cost_towers,cost_towers,cost_towers,cost_towers_thousand_dollars,cost_towers_thousand_dollars,cost_towers_thousand_dollars,cost_towers_thousand_dollars,cost_towers_thousand_dollars,cost_towers_thousand_dollars,cost_towers_thousand_dollars,cost_towers_thousand_dollars,cost_towers_thousand_dollars,cost_towers_thousand_dollars,cost_towers_thousand_dollars
-cost_chlorine_equipment_thousand_dollars,cost_chlorine_equipment,cost_chlorine_equipment,cost_chlorine_equipment,cost_chlorine_equipment_thousand_dollars,cost_chlorine_equipment_thousand_dollars,cost_chlorine_equipment_thousand_dollars,cost_chlorine_equipment_thousand_dollars,cost_chlorine_equipment_thousand_dollars,cost_chlorine_equipment_thousand_dollars,cost_chlorine_equipment_thousand_dollars,cost_chlorine_equipment_thousand_dollars,cost_chlorine_equipment_thousand_dollars,cost_chlorine_equipment_thousand_dollars,cost_chlorine_equipment_thousand_dollars
+power_requirement_mw,power_requirement_kwh,power_requirement_kwh,power_requirement_kwh,power_requirement_mw,power_requirement_mw,power_requirement_mw,power_requirement_mw,power_requirement_mw,power_requirement_mw,power_requirement_mw,power_requirement_mw,power_requirement_mw,power_requirement_mw,power_requirement_mw
```

Comment:
There's a weird thing here where, for a few years, the "power requirement (MW)" column was reported as `power_requirement_kwh`. I did some digging to confirm that this was an error and that `_kwh` should be mapped to `_mw` - is there somewhere I can document that digging for the future?